TESTING.md

```diff
@@ -0,0 +1,69 @@
+## gcloud-java tools for testing
+
+gcloud-java provides tools to make testing your application easier.
```

Comment:
The library provides tools to help with writing tests for code that uses gcloud-java services?